db/migrate/20211210215333_create_games.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateGames < ActiveRecord::Migration[6.1]
+  def change
+    create_table :games do |t|
+      t.string :game_state
+      t.string :current_player
```

Comment:
We are able to alculate these with our `ttt` gem. Is there a way we can do this with out saving these fields to the database?